test/prototype_transforms_kernel_infos.py

```diff
@@ -1449,15 +1471,16 @@ def reference_inputs_ten_crop_image_tensor():
             sample_inputs_fn=sample_inputs_five_crop_image_tensor,
             reference_fn=pil_reference_wrapper(F.five_crop_image_pil),
             reference_inputs_fn=reference_inputs_five_crop_image_tensor,
-            skips=[
-                Skip(
-                    "test_scripted_vs_eager",
-                    condition=lambda args_kwargs, device: isinstance(args_kwargs.kwargs["size"], int),
-                    reason="Integer size is not supported when scripting five_crop_image_tensor.",
+            test_marks=[
```

Comment:
We no longer need to skip `TestKernels.test_no_inplace` since there was a bug in the test.